fields/types/html/editor/modifiers/insert-atomic-block.js

```diff
@@ -1,16 +1,13 @@
 'use strict';
 
 import {
-	AtomicBlockUtils,
-	Entity,
+  AtomicBlockUtils,
```

Comment:
minor, do we need multiline here?

Reply:
It has been fixed in https://github.com/twreporter/keystone/pull/180/commits/bebfe02565ac039f5e095002eba82bb7005c9ddb.